src/cmds/scripts/pbs_snapshot

```diff
@@ -38,59 +38,30 @@
 ptllibpath=
 ptlbinpath=
 
-# Try rpm install paths
-if [ -f "/etc/debian_version" ]; then
-        ptl_prefix_lib=$(dpkg -L pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null)
-else
-        ptl_prefix_lib=$(rpm -ql pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null)
+conf="${PBS_CONF_FILE:-/etc/pbs.conf}"
+if [ -r "${conf}" ]; then
+    # we only need PBS_EXEC from pbs.conf
+    __PBS_EXEC=$( grep '^[[:space:]]*PBS_EXEC=' "$conf" | tail -1 | sed 's/^[[:space:]]*PBS_EXEC=\([^[:space:]]*\)[[:space:]]*/\1/' )
 fi
```

Comment:
we need to add else part with exit here. As pbs_conf is not readable , there is not point in proceeding further